src/main/java/com/walking/intensive/chapter1/task4/Task4.java

```diff
@@ -34,8 +34,36 @@ public static void main(String[] args) {
     }
 
     static String solveEquation(double a, double b, double c) {
-        //        Место для вашего кода
-
-        return null; // Заглушка. При реализации - удалить
+        double x1, x2;
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter1/task4/Task4.java:37:9: Each variable declaration must be in its own statement. [MultipleVariableDeclarations]